XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -23,8 +23,12 @@
     <RepositoryUrl>https://github.com/xamarin/XamarinCommunityToolkit</RepositoryUrl>
     <PackageReleaseNotes>
       
+      
```

Comment:
Maybe let's remove this section at all? it always tries to insert new empty lines

Reply:
We can do that, definitely. Once we actually have release notes on a docs portal, we can add them back in.